spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/TraceKeys.java

```diff
@@ -459,6 +459,19 @@ public void setMethodNameKey(String methodNameKey) {
 	public static class Mvc {
 
 		/**
+		 * The timestamp used to provide a concrete moment in time when request processing
```

Comment:
Yeah... the problem is that `HandlerInterceptor` is used for all types of dispatch (e.g. normal, error, etc.). So I can enter the HandlerInterceptor multiple times through request processing. That's actually a very technical detail and it indeed can confuse people :/